apps/ocp-plugin/src/components/common/WithPageLayout.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import OrganizationGuard from '@flightctl/ui-components/src/components/common/OrganizationGuard';
+
+// Restore WithPageLayoutContent when organizations are enabled for OCP plugin
```

Comment:
Commented out since last news is that there won't be support for OCP Plugin in v0.10.

In the tests I did, we only have a single organization. 
The UI behaves correctly:
 - The proxy requests have the x-flightctl-organization-id header
 - In the API logs, the "org_id" query parameter can be seen with the default organization ID.